webpack/JobWizard/JobWizard.js

```diff
@@ -18,9 +18,20 @@ export const JobWizard = () => {
 
   const setDefaults = useCallback(response => {
     const responseJob = response.data;
+    const templateValues = {};
+    const inputs = responseJob.template_inputs_with_foreign;
+    if (inputs) {
+      inputs
+        .filter(input => input.advanced)
+        .forEach(input => {
+          templateValues[input.name] = input?.default || '';
+        });
+    }
```

Comment:
I feel like this is something that the server should prepare better for the client,
so we don't need to iterate over and over

Reply:
I'm not sure what should go in the server, @adamruzicka any thoughts?

Reply:
The api we're using is used only by the wizard, so we're free to do whatever we please. In general we will have to iterate over the data somewhere to get it into the right shape and it doesn't really matter on which side it happens.

In general I would try to minimize the amount of the data that actually needs to go over the wire.

Reply:
It would be the same amount of data so I guess I can split it in 2. I will still have to pass both of the inputs to the advanced tab as the description field (https://github.com/theforeman/foreman_remote_execution/pull/555) uses the default inputs as well.

Reply:
Added a commit to split the advanced